core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -147,8 +147,9 @@ fun LpOutlinedTextFieldMail(
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun LpOutlinedTextFieldNumber(
+fun LpOutlinedTextFieldInput(
     modifier: Modifier,
+    enabled:Boolean = false,
```

Comment:
Please don't forget to review the task's acceptance criteria. (second point)

Reply:
Considered and fixed